### DISCUSSION TITLE
Webpack plugin fails in a fresh Next.js app

### DISCUSSION THREAD

**Discussion by holdenmatt** [perspective-dev/perspective#1367] [Q&A]
## Bug Report

I'm trying to use Perspective in a fresh Next.js app, and neither of the documented webpack config options work, so I'm only able to use it in INLINE mode.

I've spent a few hours trying to debug unsuccessfully, so created a clean repo to reproduce the issue.

### Steps to Reproduce:
1. Create a fresh Next.JS app using `npx create-next-app`:  https://nextjs.org/docs/api-reference/create-next-app
2. `npm install @finos/perspective @finos/perspective-webpack-plugin`
3. Add a minimal `next.config.js` file to add `new PerspectivePlugin()` as a webpack plugin.

### Expected Result:
Perspective should work without errors using the webpack plugin from the documentation.

### Actual Result:
I get 404s for psp.async.wasm and perspective.wasm.worker.js and a slew of errors.

![image](https://user-images.githubusercontent.com/764380/106786356-b1e23f00-660b-11eb-9cf6-6ce1a9bf4e9b.png)

### Environment:
![image](https://user-images.githubusercontent.com/764380/106787119-8b70d380-660c-11eb-9287-89ad1db6f053.png)

### Additional Context:

I created a clean repo here that reproduces the error:
https://github.com/holdenmatt/perspective-nextjs/blob/main/next.config.js

Appreciate any suggestions or fixes for this.

**Comment by texodus** [comment]
I'm converting this to a discussion since I'm pretty sure this is an issue configuring `nextjs` as opposed to Perspective itself.

As of `0.6.0` perspective defaults to the inline build which requires neither of these files (though makes a small trade-off in loadtime performance), so this should not be an issue anymore regardless.  [Details](https://perspective.finos.org/docs/md/js.html#webpack-plugin)

**Reply by Khalilsqu** [reply]
> I'm converting this to a discussion since I'm pretty sure this is an issue configuring `nextjs` as opposed to Perspective itself.
> 
> As of `0.6.0` perspective defaults to the inline build which requires neither of these files (though makes a small trade-off in loadtime performance), so this should not be an issue anymore regardless. [Details](https://perspective.finos.org/docs/md/js.html#webpack-plugin)

Could you please elaborate on how to configure perspective in next js. Thanks in advance.

**Comment by ankrgyl** [comment]
Was this ever resolved?
